Date: Jul 01 06:05:46 2015

From: 8b6f89ef5ec233bea5b1cd1fc395b13e826b804b Jul 02 20:26:37 2015

Subject: [PATCH] RDKRIDP-771 : supporting video sink as a waylandsink

	Need wayland and weston 1.6.0 (version)
	for gstreamer1.0-plugins-bad-1.4.4-waylandsink plugins so
	adding the recipes of version 1.6.0 which are available in
	YP Core - Fido 1.8.

Source: COMCAST

Upstream-Status: Pending

Signed-off-by: pnandyala <Pavan.Nandyala@LntTechservices.com>
---

Index: weston-1.5.0/configure.ac
===================================================================
--- weston-1.5.0.orig/configure.ac
+++ weston-1.5.0/configure.ac
@@ -503,7 +503,7 @@ if test x$wayland_scanner = x; then
 	AC_MSG_ERROR([wayland-scanner is needed to compile weston])
 fi
 
-PKG_CHECK_MODULES(WAYLAND_SCANNER, wayland-scanner)
+#PKG_CHECK_MODULES(WAYLAND_SCANNER, wayland-scanner)
 
 AC_CONFIG_FILES([Makefile src/version.h src/weston.pc])
 
